package on2013_12.on2013_12_21_Facebook_Hacker_Cup_2013_Round_3.Digits_War;

import net.egork.chelper.tester.NewTester;

import org.junit.Assert;
import org.junit.Test;

public class Main {
	@Test
	public void test() throws Exception {
		if (!NewTester.test("test/on2013_12/on2013_12_21_Facebook_Hacker_Cup_2013_Round_3/Digits_War/Digits War.task"))
			Assert.fail();
	}
}
